### PR TITLE
fat32format-gui: Add version 1.01

### PR DESCRIPTION
**Pull request by kriswilk**

### CODE REVIEW

File: bucket/fat32format-gui [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "description": "A GUI FAT32 format tool that supports volumes greater than 32 GB.",
+    "homepage": "http://ridgecrop.co.uk/",
+    "license": "Unknown",
+    "url": "http://ridgecrop.co.uk/guiformat.exe",
+    "hash": "647fb4f5108af632c3d52fec34934922c50c70585697504e92fb80b3b7d05ee3",
+    "bin": "guiformat.exe",
+    "shortcuts": [
+        [
+            "guiformat.exe",
+            "fat32format-gui"
```
Comment (Ash258) [inline bucket/fat32format-gui:12]:
```suggestion
            "FAT32 Format"
```

File: bucket/fat32format-gui [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+    "version": "1.01",
+    "description": "A GUI FAT32 format tool that supports volumes greater than 32 GB.",
+    "homepage": "http://ridgecrop.co.uk/",
+    "license": "Unknown",
+    "url": "http://ridgecrop.co.uk/guiformat.exe",
+    "hash": "647fb4f5108af632c3d52fec34934922c50c70585697504e92fb80b3b7d05ee3",
+    "bin": "guiformat.exe",
```
Comment (Ash258) [inline bucket/fat32format-gui:8]:
```suggestion
```

**Review by Ash258 — CHANGES_REQUESTED** [review]
`git mv bucket/fat32format-gui bucket/fat32format-gui.json`

**Comment by kriswilk** [comment]
> `git mv bucket/fat32format-gui bucket/fat32format-gui.json`

Sorry. 🤦‍♂️

Thanks for the other suggestions. Still new to scoop and didn't know that "bin" could be omitted in this case.

**Comment by kriswilk** [comment]
On second thought, I re-added the shim for "guiformat.exe" since it's still nice to be able to launch the tool from the command line.

**Comment by Ash258** [comment]
No it is not. If there are no command line arguments available it does not make sense to add it.

**Comment by kriswilk** [comment]
> No it is not. If there are no command line arguments available it does not make sense to add it.

OK, fair enough. I've removed it. But I have launched it more than a few times from my terminal that way.

File: bucket/fat32format-gui.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+    "version": "1.01",
+    "description": "A GUI FAT32 format tool that supports volumes greater than 32 GB.",
+    "homepage": "http://ridgecrop.co.uk/",
+    "license": "Unknown",
+    "url": "http://ridgecrop.co.uk/guiformat.exe",
+    "hash": "647fb4f5108af632c3d52fec34934922c50c70585697504e92fb80b3b7d05ee3",
```
Comment (Ash258) [inline bucket/fat32format-gui.json:7]:
```suggestion
    "homepage": "http://ridgecrop.co.uk",
    "license": "Unknown",
    "architecture": {
        "64bit": {
            "url": "http://ridgecrop.co.uk/guiformat-x64.Exe#/guiformat.exe",
            "hash": "13d045f582d559c830f2e30f835ac56803844825fe6152d2d5f78367b19edb9a"
        },
        "32bit": {
            "url": "http://ridgecrop.co.uk/guiformat.exe",
            "hash": "647fb4f5108af632c3d52fec34934922c50c70585697504e92fb80b3b7d05ee3"
        }
    },
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Comment by kriswilk** [comment]
Thought of adding the 64-bit version but was hesitant since the app author states:

> There is an x64 build of Guiformat here. However the x86 build works fine on both x86 and x64 and you should use that.

No idea why that recommendation was made but I was trying to respect it. Personally, I agree it makes sense to offer both.

**Review by Ash258 — APPROVED** [review]

**Pull request merged**